Applications/Spire/Source/UiViewer/ColorSelectorButtonTestWidget.cpp

```diff
@@ -44,6 +44,13 @@ ColorSelectorButtonTestWidget::ColorSelectorButtonTestWidget(QWidget* parent)
   create_color_button->connect_clicked_signal([=] {
     on_create_color_button_color(create_color_input->text());
   });
+  m_reset_tab_order = [=] () {
```

Comment:
Why is this a lambda expression instead of added to ```on_create_color_button_color```?

Reply:
Missed it at the time.

Reply:
I would have to duplicate the tab order code in this case, once in the lambda and again at the end of the constructor.

Reply:
Then use a member function.